vsintegration/src/FSharp.Editor/CodeFix/AddNewKeywordToDisposableConstructorInvokation.fs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
typo in Filename... should be `AddNewKeywordToDisposableConstructorInvocation.fs`